klib/kvec.h

```diff
@@ -30,11 +30,13 @@
 int main() {
 	kvec_t(int) array;
 	kv_init(array);
-	kv_push(int, array, 10); // append
+	kv_push(int, array, 10, e0); // append
```

Comment:
Well, this is somewhat subjective POV I think. Passing label into macro is an elegant way to handle rare exceptions without obscuring the rest of the code IMHO. Otherwise you need to check that condition on every nesting level as well as to have a variable declaration and appropriate if/else check in the API user as well. This is really **ugly**, not labels.  But if you prefer that, here you go. This alone adds +50 lines of code with 0 functional changes and probably have some performance impact too, although today's compilers might be able to optimize it away so that might be hard to measure.